blueprints/addon/files/.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - [main, master]
```

Comment:
This perhaps should just be `master`?

Reply:
Both makes sense to me -- new projects on GitHub start with `main`, and it is my understanding that `git` itself intends to make it the default as well.